src/Umbraco.Core/Persistence/Dtos/LanguageDto.cs

```diff
@@ -38,5 +38,15 @@ internal class LanguageDto
         [Column("mandatory")]
         [Constraint(Default = "0")]
         public bool Mandatory { get; set; }
+
+        /// <summary>
+        /// Defines the fallback language that can be used in multi-lingual scenarios to provide
+        /// content if the requested language does not have it published.
+        /// </summary>
+        [Column("fallbackLanguageId")]
+        [ForeignKey(typeof(LanguageDto), Column = "id")]
+        [Index(IndexTypes.NonClustered)]
```

Comment:
No need to have an index on this column since we never query by this column

Reply:
Removed.  Initially thought we might use if for a self-join, but wasn't needed.